transmart-core-db/grails-app/services/org/transmartproject/db/clinical/AggregateDataResourceImplService.groovy

```diff
@@ -47,18 +47,8 @@ class AggregateDataResourceImplService implements AggregateDataResource {
         User exactCountsAccessUserCopy = copyUserWithChangedPatientDataAccessLevel(user, COUNTS_WITH_THRESHOLD, COUNTS)
```

Comment:
now the user is copied twice - here and inside hidePatientCount, but it's minor

Reply:
yes, indeed. That's the price to pay to reuse the peace of code.